app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
```

Comment:
- [ ] Kindly make sure to set up associations between models, the `user` has many `posts`, `comments`, and `likes`. also don't forget to add a foreign key to `author_id`